### PR TITLE
Updated of the checker with correction to the action script

### LINKED ISSUE #25: IfcGeometricRepresentationContext
Is there a mismatch between IFC documentation and project_team sample files/checker for IfcGeometricRepresentationContext ContextIdentifier and ContextType?

### PR DESCRIPTION
**Pull request by pjanck**
## File

Fixes #25 

## Covered Concept Templates - Usages Included

*Please list all Usages you have implemented.*

1. Project Geometric Context should be correct now

## Miscellaneous

Updated the script as well to cover for two or more files changed (#31).

### CODE REVIEW

**Comment by pjanck** [comment]
Seems to work as intended: https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/actions/runs/4107268403

File: .github/workflows/schema-check.yml [hunk 1]
```diff
@@ -23,13 +23,14 @@ jobs:
         with:
           files: |
             **/*.ifc
-          files_separator: ";"
+          separator: "§"
```
Comment (SergejMuhic) [inline .github/workflows/schema-check.yml:26]: I think you can reuse this "separator" attribute. Cannot remember how. Probably by full path. 🤔

File: .github/workflows/schema-check.yml [hunk 2]
```diff
@@ -23,13 +23,14 @@ jobs:
...
       - name: Logging
         run: |
           echo 'The following files were identified for checking:' >> $GITHUB_STEP_SUMMARY
           echo '' >> $GITHUB_STEP_SUMMARY
-          IFS=';'; arrIN=(${{ steps.file_changes.outputs.all_changed_files }}); unset IFS;
-          for i in "${arrIN[@]}"; do
+          IFS='§'
```
Comment (SergejMuhic) [inline .github/workflows/schema-check.yml:31]: Why exactly do you need this 'IFS' set here?

**Review by SergejMuhic — APPROVED** [review]

File: .github/workflows/schema-check.yml (hunk 1 above)
Comment (pjanck) [reply .github/workflows/schema-check.yml:26]: Resolved in call.

File: .github/workflows/schema-check.yml (hunk 2 above)
Comment (pjanck) [reply .github/workflows/schema-check.yml:31]: Resolved in call.

**Pull request merged**